### ISSUE TITLE
Generated config modal has unreadable text

### ISSUE THREAD

**Issue by armedi** [hyperjumptech/monika-config-generator#84]
The text is unreadable because of lack of color contrast

<img width="640" alt="Screen Shot 2021-08-20 at 10 03 38" src="https://user-images.githubusercontent.com/22452017/130172820-14aee45f-51ae-4cd0-9532-a6693b1a516d.png">
